static/js/hoc/withPostList_test.js

```diff
@@ -0,0 +1,221 @@
+              sort:  "new",
+              after: afterUrl
+            }
+          }
+        )
+
+        assert.equal(inner.find("InfiniteScroll").props().hasMore, !!afterUrl)
+      })
+    })
+
+    describe("loadMore", () => {
+      let pagination
+
+      beforeEach(() => {
+        pagination = makePagination()
+      })
+
+      it("doesn't do anything if canLoadMore is not set", async () => {
+        const { inner } = await render(
+          {},
+          {
+            canLoadMore: false,
+            pagination:  pagination
+          }
+        )
+        const props = inner.find("InfiniteScroll").props()
+        await props.loadMore()
+        sinon.assert.notCalled(loadPosts)
+      })
+      ;[
+        ["?sort=some_sort&after=different_after", { sort: "some_sort" }],
+        ["", {}]
+      ].forEach(([search, expectedExtraParams]) => {
+        it(`${
+          search ? "has" : "doesn't have"
+        } query parameters from the URL for the pagination`, async () => {
+          const { inner } = await render(
+            {},
+            {
+              canLoadMore: true,
+              pagination:  pagination,
+              location:    {
+                search
+              }
+            }
+          )
+          const props = inner.find("InfiniteScroll").props()
+          await props.loadMore()
+          assert.equal(loadPosts.callCount, 1)
+          sinon.assert.calledWith(loadPosts, {
+            after: pagination.after,
+            count: pagination.after_count,
+            ...expectedExtraParams
+          })
+        })
+      })
+    })
+
+    describe("PostList", () => {
+      it("has posts", async () => {
+        const { inner } = await render()
+        const props = inner.find("PostList").props()
+
+        assert.deepEqual(props.posts, postList)
+      })
+
+      it("toggles an upvote", async () => {
+        const post = makePost()
+        const innerStub = helper.sandbox.stub()
+        const stub = helper.sandbox.stub(apiActions, "toggleUpvote")
+        stub.get(() =>
+          R.curry(async (dispatch, post) => innerStub(dispatch, post))
+        )
+        const { inner } = await render()
+        const props = inner.find("PostList").props()
+        await props.toggleUpvote(post)
+
+        assert.equal(innerStub.callCount, 1)
+        assert.equal(innerStub.firstCall.args[1], post)
+      })
+      ;[
+        ["reportPost", "showReportPost", true],
+        ["removePost", "showRemovePost", true],
+        ["reportPost", "showReportPost", false],
+        ["removePost", "showRemovePost", false]
+      ].forEach(([prop, showProp, showPropValue]) => {
+        it(`${
+          showProp ? "passes" : "doesn't pass"
+        } through ${prop} when ${String(showProp)}=${String(
+          showPropValue
+        )}`, async () => {
+          const { inner } = await render(
+            {},
+            {
+              [prop]:     prop,
+              [showProp]: showProp
```

Comment:
I don't think this test is correctly written. `showProp` is a string, and the value of the prop should be a boolean. Looks like you need to replace `showProp` with `showPropValue` for the value here and at line 174. Also, you can use `inner.find("PostList").prop(prop)` instead of `inner.find("PostList").props()[prop]`